roles/ceph-common/tasks/installs/redhat_ceph_repository.yml

```diff
@@ -31,6 +31,39 @@
     backup: yes
   when: ceph_dev
 
+- name: fetch nfs-ganesha development repo file
+  uri:
+    url: https://shaman.ceph.com/api/repos/nfs-ganesha/next/latest/{{ ansible_distribution | lower }}/{{ ansible_distribution_major_version }}/flavors/{{ nfs_ganesha_flavor }}/repo
+    return_content: yes
+  register: nfs_ganesha_dev_yum_repo
+  when:
+    - nfs_ganesha_dev is defined
```

Comment:
Why not default ``nfs_ganesha_dev`` and ``nfs_ganesha_stable`` in ``ceph-defaults/defaults/main.yml`` and avoid these ``x is defined`` checks?

Reply:
@andrewschoen that could work. What's the rule of thumb on whether to have variables in group_vars/all.yml and ceph-defaults/defaults/main.yml and one of the daemon specific main.ymls like ceph-nfs/defaults/main.yml for example?